### PR TITLE
Release notes for PS 5.5.60-38.12

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by tduzan-te** [comment]
These look okay to me.  @laurynas-biveinis probably should review before merge as well.

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: foreign keys pointing to the same table

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: The bug description does not match PS-3903, either wrong description or the ref

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: "with for"

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: Other bugs

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: For other bugs, please copy-paste their titles next to them as requested by @svetasmirnova

File: doc/source/upstream-bug-fixes.rst
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: The statuses of other bugs should be updated

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: doc/source/upstream-bug-fixes.rst
Comment (fiowro) [reply doc/source/upstream-bug-fixes.rst]: There is a ready fix for a separate DOC-734 ticket (changing launchpad links to jira ones), with updated upstream-bug-fixes.rst, and It would be convenient to bind status checks to it also.

File: doc/source/conf.py
Comment (percona-ysorokin) [inline doc/source/conf.py]: Should this start with the dash?

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst]: `Some IO_CACHE file operations were not instrumented for Performance Schema causing inaccurate statistics generated.`

**Review by percona-ysorokin — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-5.5.60-38.12.rst
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.5.60-38.12.rst:22]: s/on/to

**Pull request merged**
